vars.sample

```diff
@@ -50,10 +50,20 @@ ORACLE_TABLESPACES_DIR=/home/oracle/app/oracle/oradata
 # }}}
 
 # Only needed for ETL
+ETL_SCRIPTS=/path/to/transmart-data/samples/postgres
+export PATH=$ETL_SCRIPTS:$PATH
+
+# export KETTLE_HOME=$HOME ## kettle.properties file is expected in .kettle subdirectory of KETTLE_HOME
+
 # End with /
-KETTLE_JOBS_PSQL=/path/to/transmart-ETL/Postgres/GPL-1.0/Kettle/Kettle-ETL/
+KETTLE_JOBS_PSQL=/path/to/tranSMART-ETL/Postgres/GPL-1.0/Kettle/Kettle-ETL/
+R_JOBS_PSQL=/path/to/tranSMART-ETL/Postgres/GPL-1.0/R
```

Comment:
Is your suggestion the followoing?
PATH_TO_TRANSMART_ETL=/path/to/transmart-ETL
KETTLE_JOBS_PSQL=${PATH_TO_TRANSMART_ETL}/Postgres/GPL-1.0/Kettle/Kettle-ETL/
KETTLE_JOBS_ORA=${PATH_TO_TRANSMART_ETL}/Kettle-GPL/Kettle-ETL/
R_JOBS_PSQL=${PATH_TO_TRANSMART_ETL}/Postgres/GPL-1.0/R/